lvx frontend/src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
You can delete the file if you are not writing testcases.

Reply:
Deleted this file